packages/loader/container-loader/src/loader.ts

```diff
@@ -135,31 +135,71 @@ function createCachedResolver(resolver: IUrlResolver) {
     return cacheResolver;
 }
 
+export interface ILoaderProps {
```

Comment:
putting doxygen comments would be great!

Reply:
was just doing that :)